suite-native/accounts/src/useAccountForm.ts

```diff
@@ -0,0 +1,19 @@
+import { yup } from '@trezor/validation';
+import { useForm } from '@suite-native/forms';
+
+export const MAX_ACCOUNT_LABEL_LENGTH = 30;
+
+export type AccountFormValues = yup.InferType<typeof accountFormValidationSchema>;
+
+const accountFormValidationSchema = yup.object({
+    accountLabel: yup.string().required().max(MAX_ACCOUNT_LABEL_LENGTH),
+});
+
+export const useAccountForm = (accountName?: string) =>
+    useForm<AccountFormValues>({
+        validation: accountFormValidationSchema,
+        defaultValues: {
+            accountLabel: accountName,
+        },
+        mode: 'onChange',
```

Comment:
We shouldn't change behavior accross forms like this. Please remove this and if there is product need for onChange validation, it should be changed globally